extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -548,6 +551,14 @@ protected static function getEmailSpecificFormFields() {
 
 		return array_merge_recursive( $formFields, parent::getEmailSpecificFormFields() );
 	}
+
+	protected function findSignatureIcon() {
```

Comment:
Minor: can we rename this method to `getSignatureIcon`? "Find" makes it sound like this method would do some sort of searching, with maybe some additional logic. But we're really just calling `getFileUrl`.